packages/agent/src/database/config.ts

```diff
@@ -82,23 +71,8 @@ const postgresConfig: PostgresConnectionOptions = validatePostgresOptions({
   ssl,
   database: DB_DATABASE_NAME,
   cache: DB_CACHE_ENABLED !== 'false',
-  entities: [
-    ...VeramoDataStoreEntities,
-    ...DataStoreStatusListEntities,
-    ...DataStoreContactEntities,
-    ...DataStoreEventLoggerEntities,
-    ...DataStoreIssuanceBrandingEntities,
-    ...DataStorePresentationDefinitionEntities,
-  ],
-  migrations: [
-    ...VeramoDataStoreMigrations,
-    ...DataStoreStatusListMigrations,
-    ...DataStoreContactMigrations,
-    ...WebWalletMigrations,
-    ...DataStoreEventLoggerMigrations,
-    ...DataStoreIssuanceBrandingMigrations,
-    ...DataStorePresentationDefinitionMigrations,
-  ],
+  entities: [...VeramoDataStoreEntities, ...DataStoreEntities],
```

Comment:
I don't like this change tbh . I want to see what we  are migrating

Reply:
fixed